src/pytest_regressions/image_regression.py

```diff
@@ -43,11 +44,11 @@ def _load_image(self, filename):
         else:
             return img
 
-    def _compute_manhattan_distance(self, diff_image):
+    def _compute_manhattan_distance(self, diff_image: Any) -> float:
```

Comment:
Should `diff_image` be of type `PIL.Image`? `Any` may not have attribute `size` for example

Reply:
Same reason as in https://github.com/ESSS/pytest-regressions/pull/105#discussion_r971857637: `PIL` is an optional dependency, so we cannot really use it at the module level for type-checking.